spec/requests/protected_resources/private_api_spec.rb

```diff
@@ -73,7 +73,7 @@
   end
 
   scenario 'access token with both allowed scopes' do
-    @token.update_attribute :scopes, 'write admin'
+    @token.update_attribute :scopes, %w[write admin]
```

Comment:
I'm not sure this is the appropriate place to ensure this behavior. It would be more appropriate to have it on `spec/lib/models/scopes_spec.rb`

Reply:
I'm not found we test concern before, maybe this is a good start, I'll do this

Reply:
I meant this one: https://github.com/doorkeeper-gem/doorkeeper/blob/master/spec/lib/models/scopes_spec.rb

Reply:
oh I see